packs_add.php

```diff
@@ -118,9 +120,23 @@
 					}
 					if ( in_array( $infoPack['id'], $listPackIDs ) )
 					{
-						$listErrors[] = [ 'error_duplicate_pack_id', $infoExistingPack['id'] ];
+						$listErrors[] = [ 'error_duplicate_pack_id', $infoPack['id'] ];
 						continue;
 					}
+					if ( $infoCategory['expire'] )
+					{
+						if ( $infoPack['expiry'] == '' )
+						{
+							$listErrors[] = [ 'error_missing_expiry', $infoPack['id'] ];
+						}
+						elseif ( ! preg_match( '/^[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|' .
```

Comment:
This looks like it is validating for a date time field.  I notice that the lang files have British English and English versions, and also have some date field labels, but I didn't notice a date format prompt.  I think that would be important to include somewhere.  Sorry - I'm rubbish at reading regex, but this looks like yyyy-mm-dd hh:mm which if it is, is fine for UK/US compatibility.